<launch>
  <arg name="MANAGER" default="/camera_nodelet_manager" />
  <include file="$(find openni_launch)/launch/openni.launch">
  </include>
  <group ns="pcl_nodelet">
    <node pkg="nodelet" type="nodelet"
          name="snapit"
          args="load jsk_pcl/Snapit $(arg MANAGER)"
          output="screen" clear_params="true">
      <remap from="~input" to="/camera/depth_registered/points" />
      <rosparam>
         tolerance: 0.1
      </rosparam>
    </node>
  </group>
  <node pkg="jsk_pcl_ros"
        type="snapit_client_sample.py"
        name="snapit_sample"
        output="screen">
    <remap from="snapit" to="/pcl_nodelet/snapit/snapit" />
  </node>
</launch>
